.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: CI
+
+on: [push, pull_request_target]
+
+jobs:
+  tests:
+    if: github.event_name == 'push' || github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```

Comment:
Do we only test on 3.7 and 3.8 on CI but also 3.6 in tox?

Reply:
I removed 3.6 test cases because this library uses `dataclasses` module which is only available for 3.7+.  There is a backport but I wasn't sure how to inject this dependency into the 3.6 test environment.  I think its probably fine to not test 3.6, presumably the dataclasses backport is well tested so if 3.7/3.8 tests work so should 3.6.

Reply:
Use `dataclasses;python_version<"3.7"` in `install_requires` in `setup.py`

https://github.com/joshlk/dataclassframe/blob/3893b2ee715b93bb85560013b2c8e459cf3d8a7f/requirements.txt#L2

Reply:
Of course that's also important so that the module works on Python 3.6 for users.

Reply:
thats pretty cool, didn't know you can do that!